cli/src/fmt/simple/operation/mod.rs

```diff
@@ -0,0 +1,12 @@
+use fmt::Render;
```

Comment:
Why put this in a `mod.rs` file versus just an `operation.rs` file?

Reply:
I envisioned in the future we'd have renders in here for each of the `OperationKind`s

Reply:
I don't think it'll be as long as you think. The effects had a similar size number of variations.